third_party/libpg_query/grammar/keywords/reserved_keywords.list

```diff
@@ -34,6 +34,7 @@ FETCH
 FOR
 FOREIGN
 FROM
+FUNCTION
```

Comment:
OK I modified the FUNCTION key word so that it would work  in   SELECT FUNCTION xt().  But with your new syntax 
SELECT * FROM xt() . it will not be required.